packages/shared/src/env.ts

```diff
@@ -26,6 +26,11 @@ const EnvSchema = z.object({
   // Redis Cluster Configuration
   REDIS_CLUSTER_ENABLED: z.enum(["true", "false"]).default("false"),
   REDIS_CLUSTER_NODES: z.string().optional(),
+  REDIS_SENTINEL_ENABLED: z.enum(["true", "false"]).default("false"),
+  REDIS_SENTINEL_NODES: z.string().optional(),
+  REDIS_SENTINEL_MASTER_NAME: z.string().optional(),
+  REDIS_SENTINEL_USERNAME: z.string().optional(),
+  REDIS_SENTINEL_PASSWORD: z.string().optional(),
```

Comment:
Is there a reason why we shouldn't reuse REDIS_USERNAME and REDIS_AUTH? Does Sentinel need dedicated credentials and env variables for it to work?